### DISCUSSION TITLE
Ignoring context menu on certain DOM elements

### DISCUSSION THREAD

**Discussion by srcnalt** [shadcn-ui/ui#2750] [Q&A]
How would it be possible to ignore radix context menu and use browser's itself in certain cases? On textfields or other inputs having context menu open prevents users from copy pasting text via context menu.

**Comment by tobias-hay** [comment]
Hi, did you ever happen to figure this out?

**Reply by srcnalt** [reply]
Hi @Woofer21, haven't updated or checked it for a while. I will check and comment.

**Reply by srcnalt** [reply]
With a quick try I still am not able to get this working on my project. Might need to move some of the struture out of shadcn ui lib to get it working.
